spec/factories/city_factory.rb

```diff
@@ -5,5 +5,32 @@
     factory :valid_city do
       name { 'Львів' }
     end
+    factory :invalid_city do
+      name { nil }
+    end
+    factory :valid_name do
```

Comment:
Here and below please use naming like 'city_with_valid_name' - name of model should be in name of factory

Reply:
Ok